### DISCUSSION TITLE
Ability to Copy / Paste / Duplicate Maps

### DISCUSSION THREAD

**Discussion by RandemGamor** [AscensionGameDev/Intersect-Engine#1511] [Ideas]
**Is your feature request related to a problem? Please describe.**
Nope

**Describe the solution you'd like**
I'd like to be able to copy / paste or duplicate maps and/or their properties.
Having to manually change the properties of every map is a very tedious process.
Thanks :)

**Additional context**

**Comment by Cheshire92** [comment]
We've had a discussion about this at some point and rather than copying them we thought a more elegant solution would be to allow users to make templates of map properties.
